hydrolib/core/dflowfm/obs/models.py

```diff
@@ -37,7 +37,7 @@ class ObservationPoint(INIBasedModel):
     An observation point that is included in the observation point file.
 
     All lowercased attributes match with the observation point input as described in
-    [UM Sec.F.2.2.1](https://content.oss.deltares.nl/delft3d/manuals/D-Flow_FM_User_Manual_1D2D.pdf#section.F.2.2.1).
+    [UM Sec.F.2.2.1](https://content.oss.deltares.nl/delft3dfm1d2d/D-Flow_FM_User_Manual_1D2D.pdf#section.F.2.2.1).
```

Comment:
met een steekproef van deze link sprong het niet naar sectie F.2.2.1 toe voor mij, is dit een probleem en kan dat bij meerdere mogelijk voorkomen?

Reply:
Het gaat hier om een subsection en er wordt naar een section verwezen, mogelijk dat dat het probleem is?

Reply:
fixed in new commit